# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_header_Server_Timing.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_header_Server_Timing.js
--- a/netwerk/test/unit/test_header_Server_Timing.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_header_Server_Timing.js	Tue Feb 12 16:08:25 2019 +0000
@@ -10,7 +10,7 @@
 
 function make_and_open_channel(url, callback) {
   let chan = NetUtil.newChannel({uri: url, loadUsingSystemPrincipal: true});
-  chan.asyncOpen2(new ChannelListener(callback, null, CL_ALLOW_UNKNOWN_CL));
+  chan.asyncOpen(new ChannelListener(callback, null, CL_ALLOW_UNKNOWN_CL));
 }
 
 var responseServerTiming = [{metric:"metric", duration:"123.4", description:"description"},